app/javascript/application.js

```diff
@@ -1,4 +1,25 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
+// import "@hotwired/turbo-rails"
 import "./controllers"
 import "bootstrap"
+import { Turbo } from "@hotwired/turbo-rails";
+
+Turbo.setConfirmMethod((message, element) => {
+  const dialogButton = document.getElementById("turbo-confirm-button");
+  dialogButton.click();
+  const dialog = document.getElementById("turbo-confirm");
+  dialog.querySelector(".modal-title").textContent = message;
+
+  return new Promise((resolve) => {
+    dialog.querySelectorAll(".btn").forEach((button) => {
+      button.addEventListener(
+        "click",
+        (event) => {
+          const button = event.currentTarget;
+          resolve(button.textContent === "Confirm");
+        },
+        { once: true },
+      );
+    });
+  });
+});
```

Comment:
can provide a screenshot of what this does?

Reply:
<img width="497" alt="modal" src="https://user-images.githubusercontent.com/99309242/191934994-e3120c4e-09ce-4b5d-bd7f-158bcfcc1ad8.png">

Reply:
cool @prawnto